### PR TITLE
LPS-192117, LPS-192118, LPS-192113 | Master

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://liferay.atlassian.net/browse/LPS-192117
https://liferay.atlassian.net/browse/LPS-192118
https://liferay.atlassian.net/browse/LPS-192113

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBDisplayPage.testcase [hunk 1]
```diff
@@ -25,6 +25,47 @@ definition {
 		}
 	}
 
+	@description = "This test covers LPS-166740. It validates that the display page template can be deleted after removing the related article."
+	@priority = 4
+	test CanDeleteDisplayPageTemplateAfterRemovingReferencingArticle {
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBDisplayPage.testcase:30]: ReferencedArticle

**Review by austinchiang — CHANGES_REQUESTED** [review]
Please also include the LPS# of the bug tickets in the test description to add more traceability

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by Jeremy-Ch** [comment]
ci:forward

**Pull request closed without merging**